src/style.css

```diff
@@ -0,0 +1,144 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  --color__main: #060b35;
+  --color__secondary: #616070;
+  --color__button: #00acdc;
+  --color__border: #f3f3f3;
+}
+
+.page__card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card {
+  background-color: #fff;
+  border: 1px solid var(--color__border);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+}
+
+.card__img {
+  display: flex;
+  justify-content: center;
+}
+
+.card__icon {
+  width: 160px;
+  height: 134px;
+}
+
+.card__container {
+  display: flex;
+  flex-direction: column;
+}
+
+.card__title {
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color__main);
+  margin-bottom: 0;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color__secondary);
+  margin-top: 4px;
+  margin-bottom: 0;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+  align-items: flex-end;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price_price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price_value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color__main);
+}
+
+.card__button_buy {
+  width: 166px;
+  height: 40px;
+  margin-top: 16px;
+  background-color: var(--color__button);
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid transparent;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+  color: #fff;
+}
+
+.card__button_buy:hover {
+  background-color: #fff;
+  border: 1px solid var(--color__button);
+  color: var(--color__button);
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  list-style: none;
+  padding: 0;
+}
+
+.stars--5 :nth-of-type(n),
+.stars--4 :nth-of-type(-n + 4),
+.stars--3 :nth-of-type(-n + 3),
+.stars--2 :nth-of-type(-n + 2),
+.stars--1 :nth-of-type(1) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
```

Comment:
move each BEM block to the seperate file

Reply:
added blocks and changes in project